morango/api/viewsets.py

```diff
@@ -275,19 +271,44 @@ def create(self, request):
             "connection_path": request.data.get("connection_path"),
             "client_ip": get_ip(request) or "",
             "server_ip": request.data.get("server_ip") or "",
-            "client_instance": request.data.get("instance"),
-            "server_instance": json.dumps(
-                serializers.InstanceIDSerializer(instance_id).data
-            ),
```

Comment:
In terms of resuming sessions, we actually care more about the client/server instance_ids being the same, than we do about the IPs being the same (better indicator that the same session is likely to also exist on the other side). Since here we have the full dict (not just the instance_id), it would be hard to filter by efficiently, though. And the IPs may change, but not super frequently, so filtering by IP is probably a good middle ground. We just need to make sure we're smoothly handling cases where the session doesn't actually exist on the other side, since IPs may be reused.